src/resolvers.ts

```diff
@@ -3,15 +3,13 @@ import { User } from "./entities/user.entity";
 import { generateHash } from "./utils";
 import { CustomError } from "../src/errors";
 import { getToken } from "../src/token";
-import { UserInput } from "../src/types";
+import { UserInput, UserPayload } from "../src/types";
 
 const isValidPassword = (str: string) => {
   return /(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}/.test(str);
 };
 
-type UserInterface = {
-  name: string;
-  email: string;
+type UserID = {
```

Comment:
o nome `UserID` não explica muito o que esse tipo realmente é
```suggestion
type GraphQLContext = {
```